css-borders-4/Overview.bs

```diff
@@ -367,43 +367,42 @@ Issue <a href="https://github.com/w3c/csswg-drafts/issues/11610">#11610</a>: che
+		<dfn><<corner-shape-value>></dfn> = <l>''round''</l> | <l>''scoop''</l> | <l>''bevel''</l> | <l>''notch''</l> | <l>''<corner-shape-value>/square''</l> | <l>''squircle''</l> |
+											superellipse(<<number [-&infin;,&infin;]>> | infinity | -infinity)
 	</pre>
 
 	<dl dfn-type="value" dfn-for="<corner-shape-value>">
 		<dt><dfn>round</dfn>
 		<dd>
-			Border radii define a convex elliptical curve at the corner. Equivalent to <css>superellipse(2)</css>.
+			Border radii define a convex elliptical curve at the corner. Equivalent to <css>superellipse(1)</css>.
 
 			Note: this is the initial value of 'corner-shape' properties, as elements with 'border-radius' would be rounded.
 
 		<dt><dfn>scoop</dfn>
-		<dd>Border radii define a concave elliptical curve at the corner. Equivalent to <css>superellipse(0.5)</css>.
+		<dd>Border radii define a concave elliptical curve at the corner. Equivalent to <css>superellipse(-1)</css>.
 		<dt><dfn>bevel</dfn>
-		<dd>Border radii define a diagonal slice at the corner. Equivalent to <css>superellipse(1)</css>.
+		<dd>Border radii define a diagonal slice at the corner. Equivalent to <css>superellipse(0)</css>.
 		<dt><dfn>notch</dfn>
-		<dd>Border radii define a concave 90deg angle at the corner. Equivalent to <css>superellipse(0)</css>.
-		<dt><dfn>straight</dfn>
+		<dd>Border radii define a concave 90deg angle at the corner. Equivalent to <css>superellipse(-infinity)</css>.
+		<dt><dfn>square</dfn>
 		<dd>Border radii define a convex 90deg angle at the corner.
 			This would have the same visual effect as a 'border-radius' of 0. This is different from having a 'border-radius' of 0 when animating.
 			Equivalent to <css>superellipse(infinity)</css>.
 		<dt><dfn>squircle</dfn>
-		<dd>Border radii define a convex curve between an ellipse and an convex angle, equivalent to <css>superellipse(4)</css>.
+		<dd>Border radii define a convex curve between an ellipse and an convex angle, equivalent to <css>superellipse(2)</css>.
 	</dl>
 
-	Issue <a href="https://github.com/w3c/csswg-drafts/issues/11607">#11607</a>: resolve on ''straight'' vs <css>none</css>.
-
-	The <dfn>superellipse( <<number>> | infinity )</dfn> function describes the curvature of the corner.
-	It accepts a <dfn>superellipse exponent</dfn>, which defines the curvature of the corner, or the exponent of the formula.
-	The [=superellipse exponent=] accepts values between 0 (a straight concave angle) and <css>infinity</css> (a straight convex angle),
-	with the values in between representing the curvatures in between.
+	The <dfn>superellipse( <<number>> | infinity | -infinity )</dfn> function describes the <dfn>superellipse parameter</dfn> of the corner.
+	It is a number between <css>-infinity</css> and <css>infinity</css>, with <css>-infinity</css> corresponding to a straight concave corner,
+	<css>infinity</css> corresponding to a straight convex corner.
 
 	The ''superellipse()'' formula can be described in cartesian coordinates, as follows,
```

Comment:
I feel like I still want more explanatory text here describing how only the k > 1 part of classic superellipse function is used, and is reflected for the s < 0 cases for symmetry. I don't think the text here is enough for a from-scratch implementation, which should be the goal.